### DISCUSSION TITLE
Expose `use-delayed-hover` from Floating in @mantine/hooks

### DISCUSSION THREAD

**Discussion by islami00** [mantinedev/mantine#2499] [Feature requests]
Hi,

Thanks for creating this library!

I was making a component that needed to add a delay before it is hovered and I found the `use-delayed-hover` hook which allows something similar in `mantine-core/src/Floating/use-delayed-hover` . 

This isn't a part of the `@mantine/hooks` so I couldn't use it directly from the package, will it be possible be nice to have this hook exposed for this use case?

> Ps: I ended up combining useHover and the `use-delayed-hover` hook to get the desired effect.

**Comment by rtivital** [comment]
It is a part of floating ui, you can import any floating ui hook from this package – https://floating-ui.com/docs/react-dom-interactions

**Reply by islami00** [reply]
Thanks!
I'll import from that instead
